src/linear_algebra/std_basis.lean

```diff
@@ -251,10 +251,22 @@ by { simp only [basis_fun, basis.coe_of_equiv_fun, linear_equiv.refl_symm,
   (pi.basis_fun R η).repr x i = x i :=
 by simp [basis_fun]
 
```

Comment:
Let's end the previous sections and declare a couple of variables, to simplify the remainder:
```suggestion

end module

end pi

namespace matrix

variables (n m : Type*) [fintype m] [fintype n]
```